src/azul/project/hca/transformers.py

```diff
@@ -338,7 +327,7 @@ def create_documents(self,
             contents = dict(specimens=[_specimen_dict(s) for s in visitor.specimens.values()],
                             cell_suspensions=[_cell_suspension_dict(cs) for cs in visitor.cell_suspensions.values()],
                             files=[_file_dict(file)],
-                            processes=[_process_dict(pr, pl) for pr, pl in visitor.processes.values()],
+                            protocol=[_protocol_dict(pl) for pl in visitor.protocol.values()],
```

Comment:
```suggestion
                            protocols=[_protocol_dict(pl) for pl in visitor.protocol.values()],
```

The obvious choice for naming a variable whose value is a collection is a plural noun. What's the reason for deviating from that convention?